src/libs/API/commands/BaseAPICommandBlocking.js

```diff
@@ -0,0 +1,44 @@
+import BaseAPICommand from './BaseAPICommand';
```

Comment:
Nit picky, but I would suggest these names
- APICommand
- BlockingAPICommand
- PubSub (which has publish and susbcribe, instead of DataChanged)
- handleFailure (instead of requestFailed which sounds like a bool)
- handle666
- ValidateEmail (instead of ValidateEmailCommand -- it's in the command/ dir)